misk-cron/src/main/kotlin/misk/cron/CronModule.kt

```diff
@@ -24,7 +22,6 @@ class CronModule @JvmOverloads constructor(
   private val dependencies: List<Key<out Service>> = listOf()
 ) : KInstallOnceModule() {
   override fun configure() {
-    install(FakeCronModule(zoneId, threadPoolSize, dependencies))
```

Comment:
I’d suggest removing the MetadataModule install line from the RealMoxule instead. Less likely to cause real breakages.